public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/setQueryValue.ts

```diff
@@ -1,6 +1,8 @@
 import { AzureMetricDimension, AzureMonitorQuery } from '../../types';
 
 export function setSubscriptionID(query: AzureMonitorQuery, subscriptionID: string): AzureMonitorQuery {
+  console.log('setSubscriptionID subscriptionID:', subscriptionID, new Error().stack);
```

Comment:
are these console logs intentionally left in?

Reply:
what logs? 👁👃👁

Reply:
(i fixed them)